src/cli/build/assets.ts

```diff
@@ -0,0 +1,79 @@
+import path from 'path';
+
+import type { Color } from 'chalk';
+import chalk from 'chalk';
+import fs from 'fs-extra';
+
+import { copyFile } from '../../utils/copy';
+import { buildPatternToFilepathMap, crawlDirectory } from '../../utils/dir';
+import { type Logger, createLogger, log } from '../../utils/logging';
+import {
+  getConsumerManifest,
+  getPropFromConsumerManifest,
+} from '../../utils/manifest';
+
+export const copyAssets = async (
+  destinationDir: string,
+  logger: Logger = log,
+) => {
+  const manifest = await getConsumerManifest();
+  const assets = await getPropFromConsumerManifest<string, string[]>('assets');
+
+  if (!manifest || !assets) {
+    return;
+  }
+
+  const resolvedSrcDir = path.resolve(path.dirname(manifest.path), 'src');
```

Comment:
This is interesting as we haven't taken a strong opinion on `src` within the tooling itself, it's just the templated default. Would it make sense for `assets` to be specified relative to the parent directory (alongside `package.json`)?

Reply:
I've made the change here [`3a6232e` (#1163)](https://github.com/seek-oss/skuba/pull/1163/commits/3a6232ee4a0a18e176f22fd7fc76040e41bfda07). I think a better approach would be to use the directory from `skuba.entryPoint`. What do you think?